app/transport/http/contracts.go

```diff
@@ -16,3 +16,8 @@ type Router interface {
 	mapRoutes(UserHandler)
 	http.Handler
 }
+
+// Logger represents logger
+type Logger interface {
+	Printf(string, ...any)
+}
```

Comment:
Just asking but what if you use standard logger? Moreover we can wrap it in a handler by creating a field *log.Logger. I am not sure is this a good practice, but saw this in some examples.

Reply:
To be honest I steal code from https://dave.cheney.net/2017/01/23/the-package-level-logger-anti-pattern

Reply:
btw we were recommended to use https://github.com/uber-go/zap. Is it ok?